runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -2371,6 +2371,14 @@ TR_J9ServerVM::isForceInline(TR_ResolvedMethod *method)
    return static_cast<TR_ResolvedJ9JITServerMethod *>(method)->isForceInline();
    }
 
+bool
+TR_J9ServerVM::isPortableSCCEnabled()
+   {
+   auto stream = TR::CompilationInfo::getStream();
```

Comment:
I suggest the following code:
```
JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
auto *vmInfo = _compInfoPT->getClientData()->getOrCacheVMInfo(stream);
```
which does not use thread local storage which is more expensive to access.